src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,24 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type ToDo = {
+  todo: Todo
+};
+
+export const TodoInfo: React.FC<ToDo> = ({ todo }) => {
+  const { completed, user, title } = todo;
+
+  return user && (
+    <li className={classNames(
+      'TodoInfo',
+      {
+        'TodoInfo--completed': completed,
+      },
+    )}
+    >
+      <h2 className="TodoInfo__title">{title}</h2>
+      <UserInfo user={user} />
```

Comment:
What will be in case there will not be a user
```suggestion
      {user && <UserInfo user={user} />}
```

Reply:
I think nothing cause i used it here

![изображение](https://user-images.githubusercontent.com/92954488/213257105-a9e999ba-4097-4687-9421-bc2230955db2.png)